src/preferences/dialog/dlgprefrecord.h

```diff
@@ -2,6 +2,7 @@
 
 #include <QButtonGroup>
 
+#include "control/pollingcontrolproxy.h"
```

Comment:
Not required.
Btw you can also remove this include from the cpp and the existing ControlProxy forward-declaration here, but we need both for PollingControlProxy now, no?